tensorflow/core/kernels/mkl_conv_ops.cc

```diff
@@ -82,35 +83,41 @@ struct ConvFwdDimensions {
 };
 
 template <typename T>
-class Conv2DFwd : public DnnOp {
+class MklConv2DFwdPrimitive: public MklPrimitive {
  public:
-  explicit Conv2DFwd(const ConvFwdDimensions& convFwdDims) {
-    fwd_stream_.reset(new stream(stream::kind::eager));
+  explicit MklConv2DFwdPrimitive(const MklConvFwdParams& convFwdDims) :
+    cpu_engine_(engine(engine::cpu, 0)) {
```

Comment:
Perhaps  cpu_engine_(engine::cpu, 0) would be simple?

Reply:
Yes, I have done by (1) declaring it at bottom of class and (2) moving initialization to constructor's initialization list. Thanks!

Reply:
Sorry, this commit is for the new code. Let me double check.